client/post-editor/editor-revisions-list/index.jsx

```diff
@@ -49,13 +50,18 @@ class EditorRevisionsList extends PureComponent {
 	}
 
 	render() {
+		const usersId = uniq( map( this.props.revisions, r => get( r, 'author.ID', r.author ) ) );
```

Comment:
it's to support revisions that have already been [hydrated with author info](https://github.com/Automattic/wp-calypso/blob/master/client/state/selectors/utils/revisions/index.js#L21) (`author` is [an object](https://github.com/Automattic/wp-calypso/blob/master/client/state/data-layer/wpcom/users/index.js#L25) with `ID`, `display_name` etc ... ) and the ones that haven't (`author` is a string, containing just the ID ).

The current choice/implementation is to request fresh author information in both cases, to display the "latest" display names, but we might very well only ask for authors from revisions that haven't been updated (`author` is just an ID). I've leaned for fresh at the cost of more requests, but we can also lean in the other direction, accepting some stale-ness for less requests.

Maybe I should express it `get( r.author, 'ID', r.author )` though?